polkadot/README.md

```diff
@@ -1,13 +1,264 @@
+
+If you want to do anything on Polkadot, Kusama, or Westend, then you'll need to get an account and
+some DOT, KSM, or WND tokens, respectively. See the
+[claims instructions](https://claims.polkadot.network/) for Polkadot if you have DOTs to claim. For
+Westend's WND tokens, see the faucet
+[instructions](https://wiki.polkadot.network/docs/learn-DOT#getting-westies) on the Wiki.
+
+## Hacking on Polkadot
+
+If you'd actually like to hack on Polkadot, you can grab the source code and build it. Ensure you have
+Rust and the support software installed. This script will install or update Rust and install the
+required dependencies (this may take up to 30 minutes on Mac machines):
+
+```bash
+curl https://getsubstrate.io -sSf | bash -s -- --fast
+```
+
+Then, grab the Polkadot source code:
+
+```bash
+git clone https://github.com/paritytech/polkadot-sdk.git
+cd polkadot
+```
+
+Then build the code. You will need to build in release mode (`--release`) to start a network. Only
+use debug mode for development (faster compile times for development and testing).
+
+```bash
+./scripts/init.sh   # Install WebAssembly. Update Rust
+cargo build # Builds all native code
+```
+
+You can run the tests if you like:
+
+```bash
+cargo test --workspace --release
+```
+
+You can start a development chain with:
+
+```bash
+cargo build
+cargo run -- --dev
+```
+
+Detailed logs may be shown by running the node with the following environment variables set:
+
+```bash
+RUST_LOG=debug RUST_BACKTRACE=1 cargo run -- --dev
+```
+
+### Development
+
+You can run a simple single-node development "network" on your machine by running:
+
+```bash
+polkadot --dev
+```
+
+You can muck around by heading to <https://polkadot.js.org/apps> and choose "Local Node" from the
+Settings menu.
+
+### Local Two-node Testnet
+
+If you want to see the multi-node consensus algorithm in action locally, then you can create a
+local testnet. You'll need two terminals open. In one, run:
+
+```bash
+polkadot --chain=polkadot-local --alice -d /tmp/alice
+```
+
+And in the other, run:
+
+```bash
+polkadot --chain=polkadot-local --bob -d /tmp/bob --port 30334 --bootnodes '/ip4/127.0.0.1/tcp/30333/p2p/ALICE_BOOTNODE_ID_HERE'
+```
+
+Ensure you replace `ALICE_BOOTNODE_ID_HERE` with the node ID from the output of the first terminal.
+
+### Monitoring
+
+[Setup Prometheus and Grafana](https://wiki.polkadot.network/docs/maintain-guides-how-to-monitor-your-node).
+
+Once you set this up you can take a look at the [Polkadot Grafana dashboards](grafana/README.md) that we currently maintain. 
+
+### Using Docker
+
+[Using Docker](doc/docker.md)
+
+### Shell Completion
+
+[Shell Completion](doc/shell-completion.md)
+
+## Contributing
+
+### Contributing Guidelines
+
+[Contribution Guidelines](https://github.com/paritytech/polkadot-sdk/blob/master/docs/CONTRIBUTING.md)
+
+### Contributor Code of Conduct
+
+[Code of Conduct](https://github.com/paritytech/polkadot-sdk/blob/master/docs/CODE_OF_CONDUCT.md)
```

Comment:
broken link

```suggestion
[Code of Conduct](https://github.com/paritytech/polkadot-sdk/blob/master/polkadot/CODE_OF_CONDUCT.md)
```

Reply:
yes bc it's related to my other PR that needs to be merged